v4/source/users.yaml

```diff
@@ -39,7 +38,14 @@
                 type: string
               nickname:
                 type: string
+              auth_data:
+                description: Service-specific authentication data, such as email address.
+                type: string
+              auth_service:
+                description: The authentication service such as "email", "gitlab", or "ldap".
```

Comment:
Pretty sure this can also be `saml`

Reply:
Looks like the currently supported options are: `email`, `gitlab`, `ldap`, `saml`, `office365`, `google`, or `` (i.e. empty)